pkg/store/helper/helper.go

```diff
@@ -411,7 +411,7 @@ func (h *Helper) FetchRegionTableIndex(metrics map[uint64]RegionMetric, allSchem
 // FindTableIndexOfRegion finds what table is involved in this hot region. And constructs the new frame item for future use.
 func (*Helper) FindTableIndexOfRegion(allSchemas []*model.DBInfo, hotRange *RegionFrameRange) *FrameItem {
 	for _, db := range allSchemas {
-		for _, tbl := range db.Tables {
+		for _, tbl := range db.Tables() {
```

Comment:
we can't ensure this is the v1 path (like the HTTP API call path), should use SchemaTables

Reply:
Done